posts/2023-06-27-23.0.0.6.adoc

```diff
@@ -30,6 +30,12 @@ In link:{url-about}[Open Liberty] 23.0.0.6:
 
 Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
 
+Join our live webinars with the Open Liberty development team. Get updates and answers to your questions. The webinars are scheduled to accommodate different timezones and are similar in content. They are recorded. Register here to join live or to access the recordings later:
```

Comment:
I don't think we need the 'They are recorded.' sentence since the following one makes that clear?
```suggestion
Join our live webinars with the Open Liberty development team. Get updates and answers to your questions. The webinars are scheduled to accommodate different timezones and are similar in content.  Register here to join live or to access the recordings later:
```